reactjs/react2/week1/new-app/app/blogs/[slug].js

```diff
@@ -0,0 +1,24 @@
+// pages/blogs/[slug].js
+import { usePathname } from "next/navigation";
+
+const BlogPost = () => {
+  const pathname = usePathname();
+
+  // Extract the title from the path
+  const title = pathname.split("/").pop().replace(/-/g, " ");
+
+  // Capitalize each word in the title
+  const formattedTitle = title
+    .split(" ")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(" ");
+
+  return (
+    <div>
+      <h1>{formattedTitle}</h1>
+      <p>This is the content for {formattedTitle}.</p>
+    </div>
+  );
+};
```

Comment:
Code of the component is good, but the page doesn't work in the end.

<img width="637" alt="image" src="https://github.com/user-attachments/assets/f6aee12e-6f3f-4d87-90d0-1f6f47403e58">

Why?

Any file in `/app` folder (except _special_ ones like `page.js` or `layout.js`) are considered just components, and will be never rendered if not included to some page, and will not affect path in any way.

The only way to create a path is to create a **folder**. And inside the folder -- a `page.js`. So it suppose to be folder `[slug]` and this file suppose to be `page.js` and be inside folder `[slug]`